backend/src/AppBundle/Controller/API/ProjectController.php

```diff
@@ -1787,17 +1793,67 @@ public function putRasciAction(Request $request, Project $project, WorkPackage $
 
                 if ($request->request->get('data') != $oldRasci) {
                     $mailerService = $this->get('app.service.mailer');
-                    $mailerService->sendEmail(
-                        'MainBundle:Email:change_responsibility.html.twig',
-                        'notification',
-                        $user->getEmail(),
-                        [
-                            'username' => $user->getUsername(),
-                            'email' => $user->getEmail(),
-                            'responsibility' => $rasci->getData(),
-                            'task' => $rasci->getWorkPackageName()
-                        ]
-                    );
+                    if($oldRasci === null ) {
```

Comment:
more readable would be 
if(is_null($oldRasci)) {

Reply:
$templateName = ($oldRasci === null ) ? ''MainBundle:Email:set_responsibility.html.twig' : 'MainBundle:Email:change_responsibility.html.twig';

$mailerService->sendEmail(
                            $templateName,
                            'notification',
                            $user->getEmail(),
                            [
                                'username' => $user->getUsername(),
                                'email' => $user->getEmail(),
                                'old_responsibility' => $oldRasci,
                                'task' => $rasci->getWorkPackageName(),
                                'responsibility' => $rasci->getData(),
                                'workspace' => $worcspace,
                                'project' => $project->getName(),
                            ]
                        );